pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -575,6 +575,12 @@ func translateRouteMatch(in *networking.HTTPMatchRequest, node *model.Proxy) *ro
 		out.Headers = append(out.Headers, &matcher)
 	}
 
+	for name, stringMatch := range in.WithoutHeaders {
+		matcher := translateHeaderMatch(name, stringMatch, node)
+		matcher.InvertMatch = true
```

Comment:
@ramaraochavali negative match has been in envoy since 1.4 release right? Pretty sure it is.

Reply:
Yes. It is there

Reply:
Oh, I didn't realized that. Then I would like to ignore `withoutHeader` when `util.IsIstioVersionGE14` is false.

Reply:
I've changed it to skip that part when the version of the proxy node is less than 1.4.